scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -199,7 +199,7 @@ class Router(formatOps: FormatOps) {
         val newlineBeforeClosingCurly = decideNewlinesOnlyBeforeClose(close)
         val selfAnnotation: Option[Tokens] = leftOwner match {
           // Self type: trait foo { self => ... }
-          case t: Template => Some(t.self.name.tokens).filter(_.nonEmpty)
+          case t: Template => Some(t.self.tokens).filter(_.nonEmpty)
```

Comment:
this is because scalameta/scalameta#2334 is still not fixed.

Reply:
btw. you should be able to test out with a snapshot version of Scalameta, they are created after each merge, so we would avoid doing release that has some issues still.